### PR TITLE
Dev server user

### PR DESCRIPTION
**Pull request by Cobus-Redelinghuys**
Added UserExists functionality
May be replaced with login later

### CODE REVIEW

File: Smart-Contract-Verifier-Server/src/main/kotlin/com/savannasolutions/SmartContractVerifierServer/user/requests/GetNonceRequest.kt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.savannasolutions.SmartContractVerifierServer.user.requests
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class GetNonceRequest(@JsonProperty("UserID") val UserID: String,)
```
Comment (bitBadger8) [inline Smart-Contract-Verifier-Server/src/main/kotlin/com/savannasolutions/SmartContractVerifierServer/user/requests/GetNonceRequest.kt:5]: very nice

File: Smart-Contract-Verifier-Server/src/test/kotlin/com/savannasolutions/SmartContractVerifierServer/user/services/UserServiceTest.kt [hunk 2]
```diff
@@ -109,4 +111,45 @@ internal class UserServiceTest {
...
+    @Test
+    fun `UserExists failed due to empty wallet id`()
+    {
+        //given
+
+        //when
+        val response = userService.userExists(UserExistsRequest(""))
+
+        //then
+        assertEquals(response.status, ResponseStatus.FAILED)
```
Comment (bitBadger8) [inline Smart-Contract-Verifier-Server/src/test/kotlin/com/savannasolutions/SmartContractVerifierServer/user/services/UserServiceTest.kt:152]: much wow

**Review by bitBadger8 — APPROVED** [review]
image

**Review by Kaleb-Bruwer — APPROVED** [review]
Looks good to me

**Pull request merged**